CHANGELOG.md

```diff
@@ -3,6 +3,73 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.0
+
+_Dec 9, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/x-data-grid@v5.2.0` / `@mui/x-data-grid-pro@v5.2.0`
+
+- 🚀 Introduce the [column pinning](https://mui.com/components/data-grid/columns/#column-pinning) feature (#2946) @m4theushw
+
+  <img src="https://user-images.githubusercontent.com/5858539/145423491-bf550b77-7744-46e1-8fdb-8be1a9f5b9f5.gif" width="854" height="453" />
+
+- 🔥 Add ability to disable export options (#3270) @alexfauquette
+
+  You can disable either export options by setting `disableToolbarButton` to `true`.
+
+  ```tsx
+  <GridToolbarExport csvOptions={{ disableToolbarButton: true }} />
+  ```
+
+  ```tsx
+  <GridToolbarExport printOptions={{ disableToolbarButton: true }} />
+  ```
+
+- 🙈 Add `hideDescendantCount` option to Tree Data (#3368) @flaviendelangle
+- ⚠️ Deprecate the `getValue` param for the `valueGetter` callback (#3314) @flaviendelangle
```

Comment:
```suggestion
- ⚠️ Deprecate the `getValue` param for the `valueGetter` callback (#3314) @flaviendelangle

  Prefer accessing the `row` param:
```

@flaviendelangle do you have a diff?

Reply:
No
We could do something like

  Instead, you can access directly the `row` in the params

  ```diff
  -valueGetter: (params) => `${params.getValue(params.id, 'firstName') || ''} ${params.getValue(params.id, 'lastName') || ''}`
  +valueGetter: (params) => `${params.row.firstName || ''} ${params.row.lastName || ''}`
  ```

Reply:
Updated